configuration/csharp/http/order-processor/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+var builder = WebApplication.CreateBuilder(args);
+
+var app = builder.Build();
+
+var baseURL = (Environment.GetEnvironmentVariable("BASE_URL") ?? "http://localhost") + ":"
++ (Environment.GetEnvironmentVariable("DAPR_HTTP_PORT") ?? "3500");
+
+const string DAPR_CONFIGURATION_STORE = "configstore";
+
+var CONFIGURATION_ITEMS = new List<string> { "orderId1", "orderId2" };

```

Comment:
How common is it to pull config with multi keys?

Reply:
most SDKs have get_configuration() method that takes a list of keys as an input, so trying to show that through this example. Using a list will also show subscribing to multiple items